### DISCUSSION TITLE
Organization follower count

### DISCUSSION THREAD

**Discussion by njzjz** [lowlighter/metrics#1245] [Ideas]
Yesterday, GitHub launched [organization follower count](https://github.blog/changelog/2022-09-28-organization-follower-count-is-now-available/). I don't know if it is also available in the API, but if yes, metrics can add it to the organization profile.

**Comment by lowlighter** [comment]
Hi 👋 !

It seems it's not available yet through the GraphQL API, but will keep track of it,
Thanks for the suggestion 👍
